packages/product-feed-mergado/src/Model/FeedItem/MergadoFeedItemFactory.php

```diff
@@ -84,7 +84,7 @@ public function createForProduct(Product $product, DomainConfig $domainConfig):
             [],
             $this->availabilityFacade->isProductAvailableOnDomainCached($product, $domainId) ? 'in stock' : 'out of stock',
             $product->getBrand(),
-            $this->productUrlsBatchLoader->getResizedProductImageUrl($product, $domainConfig),
+            $this->productUrlsBatchLoader->getProductImageUrl($product, $domainConfig),
```

Comment:
Please validate with the PO whether it's ok to serve the full-sized image.
I can upload 2000x2000px image and this will be used in all feeds